gemfiles/rails_60.gemfile

```diff
@@ -0,0 +1,12 @@
+source "https://rubygems.org"
+
+eval_gemfile(File.expand_path(File.join("..", "Gemfile.common"), __dir__))
+
+gem "rails", "6.0.0.beta2"
+
+# Use the unreleased version of gems because they are compatible with Rails 6
+gem 'inherited_resources', github: 'activeadmin/inherited_resources'
+gem "activerecord-jdbcsqlite3-adapter", github: 'jruby/activerecord-jdbc-adapter', platform: :jruby
+gem 'sqlite3', '~> 1.3.6', platform: :mri, group: :test
```

Comment:
Remove this line since you have already locked the dependency in `Gemfile.common`.